validation/validation_test.go

```diff
@@ -106,7 +106,7 @@ func TestMake(t *testing.T) {
 				}),
 			},
 			expectValidator:    true,
-			expectData:         Data{A: "c"},
+			expectData:         Data{A: ""},
```

Comment:
Now validator only Bind **safe** data, it meens validation must passed, so I change some test data.

Reply:
You don't need to release this fix to v1.13.x, right? because it's a breaking change.

Reply:
> You don't need to release this fix to v1.13.x, right? because it's a breaking change.

Yes